### DISCUSSION TITLE
通过webdav连接到Alist部分功能无法使用？

### DISCUSSION THREAD

**Discussion by Mait31** [AlistGo/alist#1373] [Q&A]
在浏览器中通过alist访问百度网盘，阿里云盘下的内容可以正常访问及下载，但是通过Webdav（FE文件管理器）连接到alist后播放不了视频且下载不了，这是为什么及如何解决？

**Comment by Jue-Xu** [comment]
我好像有类似的问题
https://github.com/alist-org/alist/discussions/1402
